File: AtCoder Problems/AtCoder Beginners Selection/ABC083B - Some Sums/ABC083B - Some Sums.py
N, A, B = map(int, input().split())
ANSWER = 0

for i in range(1, N+1):
  SUM = 0
  CHECK = str(i)
  for j in range(len(CHECK)):
    SUM += int(CHECK[j])
  if A <= SUM <= B:
    ANSWER += i

print(ANSWER)

File: AtCoder Problems/AtCoder Beginners Selection/ABC083B - Some Sums/ABC083B - Some Sums.c
#include <stdio.h>

int main(){
  int N, A, B;
  scanf("%d %d %d", &N, &A, &B);

  long answer = 0;

  for (int i = 1; i <= N; i++){
    int sum = 0;
    int n = i;
    while (n > 0){
      sum += n % 10;
      n /= 10;
    }
    if (sum >= A && sum <= B){
      answer += i;
    }
  }
  printf("%ld\n", answer);
  return 0;
}